components/Headers/Header.module.css

```diff
@@ -0,0 +1,8 @@
+.welcome {
+  color: var(--accent-color-orange);
+  font-family: var(--headline-font-family);
+  font-size: 4em;
+}
+.big {
```

Comment:
I saw you're declaring and using the `.big` class for fonts and "hard-coding" fonts at several components. Maybe it would be easier in terms of descriptiveness and maintaining your code to declare those fonts styling in your GlobalStyles or an external global Typography component.

You could use more descriptive names and reference to them from everywhere. It also would be easier to change them later, as you would only need to do it at one place.

Example:
```
:root {
  /* Font Sizes */
--baseFontSize: 1rem;
--baseNavSize: 0.64rem;
--smallFontSize: 0.8rem;
--smallestFontSize: 0.512rem;
--xtra-big-heading: 3.052rem;
--xtra-heading: 2.441rem;
--heading-1: 1.953rem;
--heading-2: 1.563rem;
--heading-3: 1.25rem;
--heading-4: var(--baseFontSize);
--heading-5: var(--smallFontSize);
--heading-6: 0.64rem;
--heading-7: var(--smallestFontSize);
}
```

Reply:
> I saw you're declaring and using the `.big` class for fonts and "hard-coding" fonts at several components. Maybe it would be easier in terms of descriptiveness and maintaining your code to declare those fonts styling in your GlobalStyles or an external global Typography component.
> 
> You could use more descriptive names and reference to them from everywhere. It also would be easier to change them later, as you would only need to do it at one place.
> 
> Example:
> 
> ```
> :root {
>   /* Font Sizes */
> --baseFontSize: 1rem;
> --baseNavSize: 0.64rem;
> --smallFontSize: 0.8rem;
> --smallestFontSize: 0.512rem;
> --xtra-big-heading: 3.052rem;
> --xtra-heading: 2.441rem;
> --heading-1: 1.953rem;
> --heading-2: 1.563rem;
> --heading-3: 1.25rem;
> --heading-4: var(--baseFontSize);
> --heading-5: var(--smallFontSize);
> --heading-6: 0.64rem;
> --heading-7: var(--smallestFontSize);
> }
> ```

good idea.

Reply:
If you need some Inspiration with well scaled font sizes you'll find some here:
https://type-scale.com/